### PR TITLE
Swift Package Manager compatibility

### PR DESCRIPTION
**Pull request by TG908**
I added `Package.swift` in order to make this project usable with the swift package manager.

### CODE REVIEW

**Comment by zjfjack** [comment]
Looks perfect! Thanks for the help.
I got no time to update this project for a while.

File: README.md [hunk 1]
```diff
@@ -160,6 +160,13 @@ Horizontal scrollable range dates allow you to set your preferred scrollable ran
 
 ## Installation
 
+### Swift Package Manager
+JZCalendarWeekView can be added to your project by adding the following dependency to your `Package.swift`:
+
+```swift
+.package(url: "https://github.com/Alamofire/Alamofire.git", .upToNextMajor(from: "0.7.2"))
```
Comment (zjfjack) [inline README.md:167]: Can we change this one to JZCalendarWeekView

Comment (TG908) [reply README.md]: oops copy pasta 😄

Comment (zjfjack) [reply README.md]: One more thing, should this be 0.7.1?

Comment (TG908) [reply README.md]: well the 0.7.1 tag does not have the `Package.swift` file. This is assuming there will be a 0.7.2 release enabling swift pm

**Comment by zjfjack** [comment]
@TG908 Can we still merge this? I am thinking whether we should update Cocoapods to v0.7.2 as well?

**Comment by TG908** [comment]
Sure. The podspec should be updated as well.

**Comment by zjfjack** [comment]
Thanks @TG908

**Pull request merged**